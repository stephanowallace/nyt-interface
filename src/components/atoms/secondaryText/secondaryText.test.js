import React from 'react';
import { shallow } from 'enzyme';
import SecondaryText from '.';

describe('SecondaryText', () => {
  it('renders default state', () => {
    const component = shallow(<SecondaryText>test</SecondaryText>);
    expect(component).toMatchSnapshot();
  });
});
