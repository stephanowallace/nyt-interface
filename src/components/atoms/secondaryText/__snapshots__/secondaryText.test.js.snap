// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryText renders default state 1`] = `
<span
  className="container "
>
  test
</span>
`;
